Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -42,14 +43,24 @@ module :
 
 moduleHeader : VERSION whiteSpace numberLiteral whiteSpace? CLASS? endOfStatement;
 
+moduleConfigReferences: moduleConfigReferenceElement+;
+
+moduleConfigReferenceElement: unrestrictedIdentifier whiteSpace? EQ whiteSpace? STRINGLITERAL whiteSpace? SEMICOLON whiteSpace? STRINGLITERAL endOfStatement;
```

Comment:
I think I'd rather have literal `OBJECT` than `unrestrictedIdentifier` for the LHS part. This has potential to hide bugs or something worse if we allow for a loose interpretation.